includes/Core/Consent_Mode/REST_Consent_Mode_Controller.php

```diff
@@ -190,6 +191,29 @@ protected function get_rest_routes() {
 					),
 				)
 			),
+			new REST_Route(
+				'core/site/data/consent-api-activate',
```

Comment:
This is different to the name suggested in the IB, but I guess that's because it isn't doing the installation, only the activation, right?

Reply:
Correct, the install AJAX callback handles the main action - installing and activating the plugin. This route only activates it